layers/+lang/c-c++/packages.el

```diff
@@ -259,28 +259,23 @@
       (spacemacs/set-leader-keys-for-major-mode mode "gi" 'cscope-index-files))))
 
 ;; BEGIN LSP BACKEND PACKAGES
-
 ;; See also https://github.com/cquery-project/cquery/wiki/Emacs
 (defun c-c++/init-cquery ()
   (use-package cquery
     :if (eq c-c++-backend 'lsp-cquery)
-    :defer t
-    :commands lsp-cquery-enable
```

Comment:
@yyoncho 
Thanks for your remind. I also change my local spacemacs  config here to lsp, add (require 'cquery) in my user config. I plan to submit a PR at the moment, but feel quite happy that @MaskRay correct it so soon.